omod/src/main/webapp/pages/markPatientDead.gsp

```diff
@@ -58,6 +58,10 @@
                     jq("#cause-of-death-container > .field-error").append("${ui.message("coreapps.markPatientDead.causeOfDeath.errorMessage")}").show();
                     hasError = true;
                 }
+                if(!hasError && ${ui.convertTimezones()}){
+                    var inputDate = new Date(jq("#death-date-field").val());
```

Comment:
Is the `death-date-field` relevant only when dealing with time zones?

Reply:
It's important,  because if we are using TZ, we will convert that date to ISO format, [here](https://github.com/openmrs/openmrs-module-coreapps/pull/394/files/51b5742ddba8920005251b5200a9775ee495e384#diff-44221b510d4144f84a4334632ba5928fd06c14facbb588553a1e485117e37ea4R63).  If we are not using TZ we dont need to convert anything...